### PR TITLE
Add export handling endpoint

### PR DESCRIPTION
**Pull request by WaffelWeib**
Added 2 new endpoints for trigger and retrieve an export.
Updated 1 existing to check the report status.

FA-4095

### CODE REVIEW

File: src/Emarsys/Client.php [hunk 1]
```diff
@@ -1069,6 +1102,40 @@ protected function send($method = 'GET', $uri = '', array $body = [])
         return new Response($responseArray, (strpos($uri, '/rds') !== false && $method === 'GET'));
     }
 
+    /**
+     * send a request for given method, uri and body
```
Comment (darkmatus) [inline src/Emarsys/Client.php]: description does not match

File: src/Emarsys/Client.php [hunk 2]
```diff
@@ -1069,6 +1102,40 @@ protected function send($method = 'GET', $uri = '', array $body = [])
...
+     * @param string $method           HTTP method to use
+     * @param string $uri              path to use with basePath for sending request, defaults to basePath
+     *
+     * @return array
+     * @throws \Emarsys\Exception\ClientException
+     * @throws \Emarsys\Exception\ServerException
+     */
+    protected function sendWithCSVResponse($usedCSVDelimiter, $method = 'GET', $uri = '')
+    {
+        $headers = [
```
Comment (darkmatus) [inline src/Emarsys/Client.php:1117]: use `$this->send(...)` and handle the response as needed

Comment (WaffelWeib) [reply src/Emarsys/Client.php:1117]: The current send method expecting a json response, where this "Emarsys reponse model" is built from.
But this endpoint will return a string with the export data.
The current "send" method will not work here.
So i created a new one.

But i will change the method name to a more explicit. That it is more obvious, what it should be used for.

Comment (darkmatus) [reply src/Emarsys/Client.php:1117]: Then please change $responseJson to an other name otherwise a json should be expected

**Review by darkmatus — APPROVED** [review]

**Pull request merged**